script.js

```diff
@@ -1 +1,123 @@
+const form = document.querySelector(".input-box"),
+    input=form.querySelector("input"),
```

Comment:
```suggestion
    todoInput=form.getElementById(".todo-input"),
```

변수의 네이밍과 선택 방식이 전반적으로 모호한 것 같아요
특히 input이라고만 하면 DOM에 다른 input이 추가되었을 때 원하는 input으로 선택이 안 되는 경우도 있겠죠?!

이런 식으로 id로 input을 특정해주시는 게 좋을 것 같아요.
변수명도 구체적으로 적어주시고용!